tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -61,36 +62,36 @@ TEST(tfw_strcpy, both_plain)
 	char buf1[4] = { 0 }, buf2[4] = "abc";
 	TfwStr s1 = {
 		.len = 4,
-		.ptr = buf1
+		.data = buf1
```

Comment:
There are macros `DEFINE_TFW_STR()` and `TFW_STR_FROM()` that can be used for this purpose.